src/main/kotlin/org/rust/lang/core/resolve/RustResolveEngine.kt

```diff
@@ -150,6 +155,19 @@ object RustResolveEngine {
         return resolveIn(ResolveScopeVisitor(ref, visited), enumerateScopesFor(ref))
     }
 
+    private fun resolveImportSpec(ref: RustImportSpec, visited: MutableSet<RustUseItem>): ResolveResult {
+        val basePath = ref.parentOfType<RustUseItem>()?.viewPath?.pathPart ?: return ResolveResult.UNRESOLVED
```

Comment:
Yeah, I've never thought that `?.` chain hides not null types.

So you suggest to use something like this? Or is there a better trick? 
`val basePath = ref.parentOfType<RustUseItem>()?.let { it.viewPath.pathPart } ?: return ResolveResult.UNRESOLVED`